dali/python/nvidia/dali/ops.py

```diff
@@ -581,6 +581,53 @@ def _repack_list(self, sets, fn):
     # Operator.__call__.__doc__ = _docstring_generator_call(name)
     return Operator
 
+
```

Comment:
Plus align indentations with the rest of the file.

Reply:
Done